android/runtime/v8/src/native/V8Util.cpp

```diff
@@ -146,12 +144,41 @@ void V8Util::openJSErrorDialog(Isolate* isolate, TryCatch &tryCatch)
 		return;
 	}
 
+	HandleScope scope(isolate);
+
+	Local<Context> context = isolate->GetCurrentContext();
 	Local<Message> message = tryCatch.Message();
+	Local<Value> exception = tryCatch.Exception();
+
+	Local<Value> jsStack;
+	Local<Value> javaStack;
+
+	if (exception->IsObject()) {
+		Local<Object> error = exception.As<Object>();
+		jsStack = error->Get(context, STRING_NEW(isolate, "stack")).ToLocalChecked();
```

Comment:
This and the next line could cause a crash if the handle is empty. You could do something like this to better handle it:
```cpp
jsStack = error->Get(context, STRING_NEW(isolate, "stack")).FromMaybe(Undefined(isolate).As<Value>());
javaStack = error->Get(context, STRING_NEW(isolate, "nativeStack")).FromMaybe(Undefined(isolate).As<Value>());
```

In that code if the handle is empty due to some error, we'll default to `undefined`.